src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/ApplicationsControllerTests.cs

```diff
@@ -106,8 +106,7 @@ public async void Post_GivenValidApplication_ReturnsStatusCreatedAndCorrectData(
 
             Assert.NotNull(application);
             Assert.NotNull(application.DataTypes);
-            Assert.Single(application.DataTypes);
-            Assert.Equal("default", application.DataTypes[0].Id);
+            Assert.Empty(application.DataTypes);
```

Comment:
I see we're using `ReadAsStringAsync().Result` above, could you change that to await? Look for more cases and fix as you see fit.